components/CurrentPet.jsx

```diff
@@ -1,10 +1,13 @@
 import StyledList from "@/components/Styles/StyledList";
 import StyledListItem from "@/components/Styles/StyledListItem";
+import { SVGIcon } from "@/utils";
 
 export default function CurrentPet({ selectedPet }) {
   return (
     <StyledList>
-      <StyledListItem $variant="select">{selectedPet.type}</StyledListItem>
+      <StyledListItem $variant="select">
+        <SVGIcon variant={selectedPet.type} size={300} />
```

Comment:
Probably better to call props names starting with "$" else you are passing useless attributes to the dom